azure-quantum/tests/unit/test_job_results.py

```diff
@@ -46,8 +46,8 @@ def test_job_get_results_with_expired_sas_token(self):
     def test_job_for_microsoft_quantum_results_v1_success(self):
         job_results = self._get_job_results("microsoft.quantum-results.v1","{\"Histogram\": [\"[0]\", 0.50, \"[1]\", 0.50]}")
         self.assertTrue(len(job_results.keys()) == 2)
-        self.assertTrue(job_results["[0]"], 0.50)
-        self.assertTrue(job_results["[1]"], 0.50)
+        self.assertEqual(job_results["[0]"], 0.50)
+        self.assertEqual(job_results["[1]"], 0.50)
```

Comment:
good catch! could you please also do the same for all such bugs in this file like on line 56?